docker/Dockerfile

```
@@ -7,10 +7,7 @@ WORKDIR /usr/src/app
 COPY requirements.txt .
 
 RUN apt update && apt install -y python3-pip \
-    && pip3 install --no-cache-dir -r requirements.txt \
-    && apt remove -y python3-pip \
-    && apt autoremove --purge -y \
-    && rm -rf /var/lib/apt/lists/* /etc/apt/sources.list.d/*.list
+    && pip3 install --no-cache-dir -r requirements.txt
```

It's ok to remove the apt cache too.

However I suggest to keep the .list files.

```suggestion
    && rm -rf /var/lib/apt/lists/* \
    && pip3 install --no-cache-dir -r requirements.txt
```